datacats/cli/pull.py

```diff
@@ -38,7 +38,22 @@ def pull(opts):
                      you are going offline.
 """
     for i in IMAGES + (EXTRA_IMAGES if opts['--all'] else []):
-        pull_image(i)
+        retrying_pull_image(i)
+
+
+def retrying_pull_image(image_name):
+    while True:
```

Comment:
```python
for retries in range(1, 6):
```
would let you remove the `retries = 0` and `retries += 1` and it also has the benefit of actually stopping after 5 retries ;-)